tests/rbd/rbd_utils.py

```diff
@@ -141,11 +142,18 @@ def create_image(self, pool_name, image_name, size, **kw):
 
     def set_ec_profile(self, profile):
         self.exec_cmd(cmd="ceph osd erasure-code-profile rm {}".format(profile))
-        self.exec_cmd(
-            cmd="ceph osd erasure-code-profile set {} k={} m={}".format(
-                profile, self.k_m[0], self.k_m[2]
+        if self.failure_domain == "osd":
```

Comment:
can you please explain about this check in comment?

Reply:
this check we have introduce to create EC pool with different k_m values, Basically by default failure_domain is set to 'Node'. So suppose if we have k_m values as 3_2 then we need 5 osd node in cluster.
With this change if user passed failure_domain as 'osd', then we will check for 5 osd in cluster, and this will create a healthy cluster and active+clean PG. So ultimatly no need to add more nodes in cluster for testing.